roles/openshift_storage_cinder/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Setup RDO repositories
+  yum: name=https://rdoproject.org/repos/rdo-release.rpm state=present
```

Comment:
This would have to be conditional, we would only want to configure this repo when deployment_type == 'origin'